server/vmap/trillian_map_server/main.go

```diff
@@ -34,15 +34,15 @@ var privateKeyFile = flag.String("private_key_file", "", "File containing a PEM
 var privateKeyPassword = flag.String("private_key_password", "", "Password for server private key")
 
 func checkDatabaseAccessible(registry extension.Registry) error {
```

Comment:
I've went and done most of it, but then I get tests failing due to the magical logID zero not being there. This will be easier to do after a few PR, so I've added a TODO for now.